### DISCUSSION TITLE
posts/faas

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#6] [General]
# posts/faas/

a student obsessed with the computing world

https://whexy.com/posts/faas/

**Comment by whexy** [comment]
也欢迎评论~ 现在评论插件从 [utterances](https://utteranc.es/) 换成 [giscus](https://giscus.app/) 了。giscus 本来就支持给文章做出反应，比如点赞、踩、诧异等等，但仍旧需要 GitHub 登录。为了交互方便，我关闭了做出反应的功能。
